src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2982,6 +2984,48 @@ void itemCheckOutShouldNotAffectRequestAssociatedWithAnotherItemOfInstance() {
       is(RequestStatus.OPEN_NOT_YET_FILLED.getValue()));
   }
 
+  @Test
+  public void itemCheckOutRecallRequestCreatedShouldProduceNotices() {
```

Comment:
The test verifies reception of a notice, perhaps its better to reflect the same in the title (..Notice instead of ..Notices)?

Reply:
refactored